static/docs/style.css

```diff
@@ -17,6 +17,10 @@ table.def th {
   width: auto !important;
 }
 
+table.def th, table.def td {
+    text-align: left;
+}
+
```

Comment:
We want to respect what the original `align` attribute was, instead of setting all to left.
Combined with https://github.com/marcoscaceres/respec.org/pull/129/files#r511496878, we would ideally want 3 class names: `align-right`, and `align-center`.